### PR TITLE
chore(repo): update readme and links

### PR DESCRIPTION
**Pull request by tknickman**
### Description

Update the README, and update repository links

### CODE REVIEW

**Comment by dimitropoulos** [comment]
`crates/turborepo-lib/src/daemon/proto/turbod.proto` has the old path.  I think we should update it to prevent downstream dependencies having build errors (or `go get` failing)

File: packages/turbo-utils/__tests__/examples.test.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe("examples", () => {
   describe("getRepoInfo", () => {
```
Comment (dimitropoulos) [inline packages/turbo-utils/__tests__/examples.test.ts:39]: not sure if we care, but 5 lines above this there's a /vercel/turbo (in a non-github.com test)

**Comment by dimitropoulos** [comment]
it still works ok after the rename but we may want to also update `packages/turbo-utils/src/examples.ts` in `downloadAndExtractExample`

**Comment by tknickman** [comment]
Done done, good catches - thanks!

**Comment by dimitropoulos** [comment]
the `CONTRIBUTING.md` has two references to the old path, but since I'm about to overhaul that anyway it's probably not worth it (up to you! :smile: )

**Review by dimitropoulos — APPROVED** [review]

**Comment by tknickman** [comment]
Hmm I don't see those in Contributing - but yea feel free to fix those up in your PR!

**Pull request merged**